.github/workflows/ci-actions.yml

```diff
@@ -0,0 +1,34 @@
+name: Arquillian Weld Container CI
```

Comment:
Any particular reason for GH action vs Circle CI? It's good to see `.travis.yml` problem being solved*, I just wonder if we should strive for having a unified CI view and maintenance for the whole org (with the downside being shared quotas for example) or leave freedom to choose for every project.


*  on a personal level I'm a bit sad about what happened to this milestone in CI space after acquisition though :(

Reply:
Good point, I didn't even think about it. I used GH actions throughout Weld projects so I automatically added it here as well.

I am only familiar with Travis (whose free version was a joke last time I checked) and with GH Actions which has fairly astonishing throughput and limits for free usage. So I am biased towards GH actions :-)
I don't know CircleCI and what ups/downs it has but if you prefer it, I can change it - your call.

Reply:
I checked the how-to setup for CircleCI but it seems it requires admin to set it up on their website which I cannot do either way. But maybe there is another way I missed.

@bartoszmajsak let me know how you want to proceed - if you are fine with GH actions, I think we can merge it as-is. If you want CircleCI, I can withdraw GH actions changes from this PR.

Reply:
I think we can stick with it if that works for you. Gonna push the release buttons today :)